source/conf.py

```diff
@@ -82,6 +82,7 @@
 .. _oneAPI:  https://oneapi.com
 .. _SYCL: https://www.khronos.org/sycl/
 .. |SYCL_SPEC| replace:: `SYCL Specification`_
+.. |SYCL_SPEC_HEADER_FILES| replace:: SYCL Specification `Section 4.3 <https://registry.khronos.org/SYCL/specs/sycl-2020-provisional.pdf#page=56&zoom=100,96,189>`__
```

Comment:
There will be questions about the Specification Reference
Let's align with SYCL WG